conf/janus.jcfg.sample.in

```diff
@@ -423,9 +423,13 @@ loggers: {
 # especially if you have many PeerConnections active. To change this,
 # just set 'stats_period' to the number of seconds that should pass in
 # between statistics for each handle. Setting it to 0 disables them (but
-# not other media-related events).
+# not other media-related events). By default janus sends single media
+# statistic events per media (audio, video and the simulcast layers each
+# as one event.) If you prefer to receive one event containing all media
+# stats in one event set combine_media_stats to true.
```

Comment:
There's "one event" twice in this sentence. I'd simplify it as:

    By default janus sends single media statistic events per media (audio, video and simulcast layers as separate events): if you'r rather receive a single containing all media stats in a single array, set 'combine_media_stats' to true.